stations.csv

```diff
@@ -7148,7 +7148,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 9016;Labatut;labatut;8767230;;43.5500000000;-0.9833330000;;FR;Europe/Paris;f;f;f;f;f;f;FRGFK;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 9017;Villefranche-d’Allier;villefranche-dallier;8733145;;;;;FR;Europe/Paris;f;f;f;f;f;f;FRVFA;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 9018;Sames—Guiches;sames-guiches;;;;;;FR;Europe/Paris;f;f;f;f;f;f;FRGFM;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;10214;;;;;;;;;;;;;;;;;;
-9019;St-Denis;st-denis;;;48.9333330000;2.3666670000;;FR;Europe/Paris;t;f;f;f;f;f;FRSNS;;f;;f;;f;u09wmy;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;サン＝ドニ;생드니;;;Сен-Дени;;;聖但尼
```

Comment:
Why we need to remove the busbud ID?

Reply:
I put it in a new station.

So, as we discussed, the new station is removed, and Busbud & Flixbus ID are set at the city level